app/routes/application.js

```diff
@@ -104,6 +113,8 @@ export default Route.extend(ApplicationRouteMixin, {
                 setShowMetadata(stream, show)
               });
             }
+          } else {
+            clearShowMetadata(stream);
           }
```

Comment:
something feels weird to me about this, like i want to say we could just call clearShowMetadata at the top every time and then set on top that if we need to instead of another conditional,  but the async findRecord call here means that this wouldn't actually be the same since there could be a delay, so we do need to do it how it is here. I think this is good.